package.json

```diff
@@ -23,6 +23,7 @@
     "lint:docs:rule-examples": "node Makefile.js checkRuleExamples",
     "lint:fix": "node Makefile.js lint -- fix",
     "lint:fix:docs:js": "node Makefile.js lintDocsJS -- fix",
+    "lint:imports": "knip",
```

Comment:
I also think it'd be good to talk about running Knip in CI in some way. I personally run it for all commits in my repos.

Reply:
Agreed. We could add a new job in `ci.yml` for this.